demo/virtual-node.yaml

```diff
@@ -0,0 +1,188 @@
+apiVersion: v1
+kind: Node
+metadata:
+  annotations:
+    kwok.x-k8s.io/node: fake
+  labels:
+    beta.kubernetes.io/arch: amd64
+    beta.kubernetes.io/os: linux
+    gpu_affinity_capable: "true"
+    healthcolor: green
+    kubernetes.io/arch: amd64
+    kubernetes.io/hostname: virtual-dgxa100.40g-0
+    kubernetes.io/os: linux
+    kubernetes.io/role: agent
+    node-role.bcp.ngc.nvidia.com/role: user-workload
+    node-role.kubernetes.io/agent: ""
+    nodeGroup: gpu
+    nodeInstance: dgxa100.40g.8.norm
+    nodeType: gpu
+    pod_gpu_size.static: dynamic
+    preempt_upon_cordon: "true"
+    type: kwok
+  name: virtual-node
+spec:
+  taints: # Avoid scheduling actual running pods to fake Node
+  - effect: NoSchedule
+    key: kwok.x-k8s.io/node
+    value: fake
+  podCIDR: 10.244.3.0/24
+  podCIDRs:
+  - 10.244.3.0/24
+status:
+  allocatable:
+    cpu: "256"
+    ephemeral-storage: 15Ti
+    hugepages-1Gi: "0"
+    hugepages-2Mi: "0"
+    intel.com/mlnx_sriov_rdma1: "1"
+    intel.com/mlnx_sriov_rdma2: "1"
+    intel.com/mlnx_sriov_rdma3: "1"
+    intel.com/mlnx_sriov_rdma4: "1"
+    intel.com/mlnx_sriov_rdma5: "1"
+    intel.com/mlnx_sriov_rdma6: "1"
+    intel.com/mlnx_sriov_rdma7: "1"
+    intel.com/mlnx_sriov_rdma8: "1"
+    memory: 1Ti
+    nvidia.com/gpu: "8"
+    pods: "110"
+  capacity:
+    cpu: "256"
+    ephemeral-storage: 15Ti
+    hugepages-1Gi: "0"
+    hugepages-2Mi: "0"
+    intel.com/mlnx_sriov_rdma1: "1"
+    intel.com/mlnx_sriov_rdma2: "1"
+    intel.com/mlnx_sriov_rdma3: "1"
+    intel.com/mlnx_sriov_rdma4: "1"
+    intel.com/mlnx_sriov_rdma5: "1"
+    intel.com/mlnx_sriov_rdma6: "1"
+    intel.com/mlnx_sriov_rdma7: "1"
+    intel.com/mlnx_sriov_rdma8: "1"
+    memory: 1Ti
+    nvidia.com/gpu: "8"
+    pods: "110"
+  conditions:
```

Comment:
It looks like a lot of conditions are filled in by internal components.

Reply:
Yes, so it's a real  node configuration.